config/routes/graphql.rb

```diff
@@ -1,3 +1,3 @@
-get "/graphql", to: "graphql#query"
-post "/graphql", to: "graphql#query"
+post "/graphql", to: "graphql#execute"
+get "/graphql", to: "graphql#execute"
```

Comment:
could the removal of the `get` be a problem regarding backwards compatibility somewhere in the world?

Reply:
`get` wasn't removed as you can see above.
I removed it in one commit, then added it back again

Reply:
sure! my fault